packages/app-project/pages/_app.js

```diff
@@ -104,19 +103,17 @@ function MyApp({ Component, pageProps }) {
         <GlobalStyle />
         <Provider store={store}>
           <ThemeModeContext.Provider value={themeContext}>
-            <MediaContextProvider disableDynamicMediaQueries>
```

Comment:
This was only used for GenericAnnouncement, so it's safe to remove from `_app.js` and `_document.js`